FirebaseAppCheck/Tests/Unit/Core/FIRAppCheckTests.m

```diff
@@ -38,6 +38,10 @@
 
 #import "FirebaseCore/Extension/FirebaseCoreInternal.h"
 
+// Because the DeviceCheck provider is the default provider for AppCheck. There
+// may be test cases that are dependent on DeviceCheck being available.
```

Comment:
super nit: Maybe something like 
> Since DeviceCheck is the default attestation provider for AppCheck, disable test cases that may be dependent on DeviceCheck being available.

I think changing the period to a comma would also totally work too 😀

Reply:
Done in d7bff0b